Gems/Atom/RHI/Vulkan/Code/Source/RHI/Conversion.cpp

```diff
@@ -1154,15 +1154,20 @@ namespace AZ
                 return RHI::CheckBitsAny(usagesAndAccesses.front().m_access, RHI::ScopeAttachmentAccess::Write) ? VK_IMAGE_LAYOUT_DEPTH_STENCIL_ATTACHMENT_OPTIMAL : VK_IMAGE_LAYOUT_DEPTH_STENCIL_READ_ONLY_OPTIMAL;
             case RHI::ScopeAttachmentUsage::Shader:
             case RHI::ScopeAttachmentUsage::SubpassInput:
-                // If we are reading from a depth/stencil texture, then we use the depth/stencil read optimal layout instead of the generic shader read one.
-                if (RHI::CheckBitsAny(imageAspects, RHI::ImageAspectFlags::DepthStencil))
                 {
-                    return RHI::CheckBitsAny(usagesAndAccesses.front().m_access, RHI::ScopeAttachmentAccess::Write) ? VK_IMAGE_LAYOUT_GENERAL : VK_IMAGE_LAYOUT_DEPTH_STENCIL_READ_ONLY_OPTIMAL;
+                    // If we are reading from a depth/stencil texture, then we use the depth/stencil read optimal layout instead of the generic shader read one.
+                    // Note that if the Image is ShaderWrite we always set VK_IMAGE_LAYOUT_GENERAL, even in a read scope.
```

Comment:
This comment is a little confusing as I am not sure what is a read scope in this context. Maybe something like this is more clear->

Use VK_IMAGE_LAYOUT_GENERAL for Write usage or if the view is bound for ShaderWrite